corehq/apps/domain/templates/domain/partials/project_version_display.html

```diff
@@ -21,7 +21,8 @@ <h3><button class="btn btn-danger" type="submit">{% trans 'Unpublish' %}</button
                    data-action="{% url "domain_set_published" domain version.name %}"
                    data-toggle="modal" data-target="#contentDistributionAgreement"
                    href="#contentDistributionAgreement"
-                   onclick="hqImport('analytix/js/google').track.event('Exchange', 'Publish Most Recent', '{{ version.name }}');"
+                   data-analytics = "{versionName:{{ version.name }}}"
```

Comment:
Why not just have `data-version-name="{{ version.name }}"` instead of an object?